test/scalafmt/formatted-test.scala

```diff
@@ -0,0 +1,8 @@
+package scalarules.test.scalafmt
+import org.scalatest._
+class FormatTest extends FlatSpec {
+  val message = "We will format this test!"
+  "FormatTest" should "be formatted" in {
+    assert(message.equals("We will format this test!"))
```

Comment:
the content doesn't matter, right?

Reply:
Right. The content doesn't matter.

Reply:
I'd consider removing it for a "success" case mainly since it can surprise others. Having said that I don't feel strongly about it so your decision

Reply:
> removing it for a "success" case

Not sure what exactly you mean here. But I can see the dummy string could be confusing. I change it to `assert(true)`, it should be more obvious.

Let me know if I misunderstand.

Reply:
Sounds great (In specs2 you can have the body contain just `success`)